### PR TITLE
Add thirdparty/ ubsan suppressions for TPS demo

### PR DESCRIPTION
**Pull request by hpvb**
With these suppressions it is possible to import and run the TPS demo without any thirdparty/ reported ubsan.

This is useful because it is a complex project, so it is good for testing if new errors show up in our own code.

Sorry for the double PR, I could've rolled this into the previous one, but I didn't think to do this until now. This should be the last of it though.

### CODE REVIEW

File: misc/error_suppressions/ubsan.txt [hunk 1]
```diff
@@ -19,14 +43,19 @@ implicit-integer-sign-change:thirdparty/mbedtls/library/base64.c
...
 implicit-integer-sign-change:thirdparty/recastnavigation/Recast/Include/Recast.h
 implicit-integer-sign-change:thirdparty/recastnavigation/Recast/Source/RecastMesh.cpp
+implicit-integer-sign-change:thirdparty/spirv-reflect/spirv_reflect.c
 implicit-integer-sign-change:thirdparty/thorvg/src/renderer/sw_engine/tvgSwRaster.cpp
 implicit-integer-sign-change:thirdparty/tinyexr/tinyexr.h
 implicit-integer-sign-change:thirdparty/vulkan/vk_mem_alloc.h
+implicit-integer-sign-change:thirdparty/xatlas/xatlas.cpp
 implicit-integer-sign-change:thirdparty/zlib/trees.c
 implicit-integer-sign-change:thirdparty/zstd/compress/zstd_cwksp.h
-integer-divide-by-zero:thirdparty/misc/ok_color.h
```
Comment (akien-mga) [inline misc/error_suppressions/ubsan.txt:27]: Did you mean to remove this one?

Comment (hpvb) [reply misc/error_suppressions/ubsan.txt:27]: Yeah, it supposed to be float, not int. It is added back in as well.

Comment (hpvb) [reply misc/error_suppressions/ubsan.txt:27]: Actually there's another case of this. I'll need to do another force push. It seems that github ci sometimes decides to not give me the latest logs...

Comment (hpvb) [reply misc/error_suppressions/ubsan.txt:27]: Okay, this is definitely good now. I checked the github ci, and ran the same test locally, and the output matches.

**Comment by hpvb** [comment]
I think this can be merged, there's no more `third-party/` if you look in the ci output of this commit!

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**